static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -149,7 +148,7 @@ function IssueListHeader({node}: {node: TraceTreeNode<TraceTree.NodeValue>}) {
       : isAutogroupedNode(node)
         ? node.performance_issues.length
         : isTraceErrorNode(node)
-          ? 1
+          ? 0
```

Comment:
This is too many ternary operators for me to grok, but if its `? 0 : 0` at the end isn't that redundant?

Reply:
Yup, I should actually close this as @Abdkhan14 tackled this in a larger refactor

Reply:
Good eyes @wmak 🥇